packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -316,4 +327,12 @@ export class OutgoingPaymentEvent extends WebhookEvent {
       throw new Error(OutgoingPaymentEventError.OutgoingPaymentIdRequired)
     }
   }
+
+  $formatJson(json: Pojo): Pojo {
```

Comment:
Just wondering, since we found that the issue with date formatting was surfacing from the db level rather than the in-code model, is this `$formatJson` necessary or does ObjectionJS automatically call the `$formatJson` method of any nested models?